Comida perecedera / Fertilidad zonas

AxelPriet opened the pull request:
Se agregaron zonas de fertilidad que se pueden configurar y aumentar antes de iniciar la simulación, adicionalmente la el feature de comida perecedera que hace que la comida desaparezca después de cierto tiempo.

AxelPriet committed:
Creacion Scirpts

Se creo el srcript correspondiente a los features de fertilidad en zonas y comida perecedera

AxelPriet committed:
Scripts base y nuevo script

Se añadió la funcionalidad base de los features y se añadio uno nuevo llamado Fertilidad Food spawner

AxelPriet committed:
Zonas fertiles

Se agrego la funcionalidad de zonas fértiles por colores dependiendo la misma, si la fertilidad es menor a 5 se vera amarilla en el inspector y si es mayor se vera verde

AxelPriet committed:
Zonas Implementadas

Draklif commented on Assets/Scripts/ComidaPerecedera.cs:
Cambiar por Simulate()

AxelPriet committed:
Corrección Simulate

Se agrego que la comida perecedera aparezca atravez del SimulateManager, por un metodo Simulate
